std_msgs/Header header
uint16 color
uint16 x_pixel
uint16 y_pixel